### ISSUE TITLE
Create guide for Nether Roof map

### ISSUE THREAD

**Issue by TechnicJelle** [BlueMap-Minecraft/BlueMapWiki#73]
1. inside your `BlueMap/maps/` folder, there should be a `world_nether.conf` file (or similarly named). duplicate that, and call the other one `world_nether_roof.conf` (or similar)
2. inside the normal `world_nether.conf` file, set `render-edges` to `false`, and set `max-y` to `126`
  this makes it possible to see _through_ the roof, into the cavey nether inside, and it prevents rendering the stuff on the roof
3. inside the duplicated `world_nether_roof.conf` file, set the `name` to `"Nether Roof"`, so you can make out the different maps on your BlueMap website
4. uncomment (remove the `#` in front of) and set the `min-y` to `127`, and comment out the `max-y` (put a `#` in front of it
  this makes sure only the nether roof is rendered, and not anything else
5. do `/bluemap reload` and then do `/bluemap purge world_nether`, to rerender the normal nether map with your newly changed settings

now both of your nether maps should be rendering properly
you can check their progress by running `/bluemap`
you can also go onto the bluemap website already, and click the `Update Map` button in the menu to see the rendered areas

**Comment by TechnicJelle** [comment]
Gotta clean this up a bit, and then create an actual file for it in the community wiki

**Comment by MightyKnight** [comment]
I suggest that you seperate the guide for the Nether roof and the other variation of a Nether map. They don't belong together.

Because I was really confused at step 2, setting all the values, until I realised that it would change my existing map.
The default Bluemap Nether is fine for most users and I think it looks better than the one without `render-edges`.

**Comment by TechnicJelle** [comment]
@MightyKnight Fair point. I'll make note of that for the final version of the guide. I probably will split them apart at least a little bit.
Maybe make the modification of the original map and optional step...

**Comment by TechnicJelle** [comment]
Todo: double check compatibility with BlueMap 5
